web-app/js/portal/prototypes/OpenLayers.js

```diff
@@ -119,6 +119,7 @@ OpenLayers.Layer.WMS.buildGetFeatureRequestUrl = function(baseUrl, layerName, ou
     wfsUrl += '&outputFormat=' + outputFormat;
     wfsUrl += '&REQUEST=GetFeature';
     wfsUrl += '&VERSION=1.0.0';
+    wfsUrl += '&maxFeatures=50';
```

Comment:
Yes this is what it defaults to through the UI so that a user doesn't inadvertently ask to download the whole collection, but its not what we would want to use in the portal.   The maxfeatures parameter limits the number of records (features) returned in the response to the specified number i.e. it truncates the response.  We've just recently upped our server limit from 100000 records because that was too small so 50 records is going to drastically reduce this.